articles/vmware/vmw-faq-synchronous-protection.md

```diff
@@ -13,17 +13,17 @@ toc_fullpath: FAQs/vmw-faq-synchronous-protection.md
 toc_mdlink: vmw-faq-synchronous-protection.md
 ---
 
-# Synchronous Protection FAQa
+# Synchronous Protection FAQs
 
 ## General
 
 ### What is the ENHANCED service?
 
-ENHANCED was a G-Cloud 7 service level option which has now been replaced with a POWER VM with Synchronous Protection and Geo-resilient storage.
+ENHANCED was a G-Cloud 7 service level option which was subsequently replaced with the product name "Synchronous Protection". It has now been replaced with [Journaling Protection](vmw-sco-journaling-protection.md).
 
```

Comment:
not strictly true. Synchronous protection is still available - Journaling is the preferred alternative.